e2e/.gitignore

```diff
@@ -0,0 +1,7 @@
+
+
+/vendors
```

Comment:
this probably isn't needed since the name of the dir you're probably trying to exclude is just called "vendor" and it's already excluded from the repo's root `.gitignore` file